Hodge_Zach/assignments/bubble_sort/bubble_sort.py

```diff
@@ -0,0 +1,25 @@
+import random
+
+a = []
+
+for i in range(100):
+    a.append(random.randint(0, 10000))
+
+print a
+
+def bubble_sort(lst):
+    temp = 0
+    end = len(lst)
+
+    while end != 0:
+        for i in range(end):
+            if i != (end - 1) and lst[i] > lst[i + 1]:
+                temp = lst[i]
```

Comment:
Overall, good.
Please also re-review the reading on Tuples to see the Pythonic method of swapping variables using only a single line of code.